playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from "@playwright/test"
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
unintended comment?

Reply:
this was autogenerated by playwright. i left it in here cos it might be useful next time but can always remove if you feel like it's extra